pennylane/ops/op_math/condition.py

```diff
@@ -547,7 +564,7 @@ def new_wrapper(*args, **kwargs):
             elifs_conditions.append(pred)
             jaxpr_elifs.append(jax.make_jaxpr(functools.partial(elif_fn, **kwargs))(*args))
 
-        conditions = jax.numpy.array([condition, *elifs_conditions, True])
+        conditions = [condition, *elifs_conditions, True]

```

Comment:
:+1:

Reply:
This was changed because if any conditions are `MeasurementValue`s, we cannot put them in jax arrays as they are not a numeric type.

Reply:
Cool. I prefer this version!